app/navigation/ItemsNavigator.scala

```diff
@@ -40,6 +52,17 @@ class ItemsNavigator @Inject() extends BaseNavigator {
   }
 
   private[navigation] val checkRouteMap: Page => UserAnswers => Call = {
+    case ItemExciseProductCodePage(idx) => (answers: UserAnswers) =>
+      answers.get(ItemExciseProductCodePage(idx)) match {
+        case Some("S500" | "T300" | "S400" | "E600" | "E800" | "E910") =>
+          //TODO: Route to CAM-ITM43 when implemented
+          testOnly.controllers.routes.UnderConstructionController.onPageLoad()
+        case Some(_) =>
+          //TODO: Route to CAM-ITM38 when implemented
+          testOnly.controllers.routes.UnderConstructionController.onPageLoad()
+        case _ =>
+          controllers.sections.items.routes.ItemsIndexController.onPageLoad(answers.ern, answers.draftId)
+      }
```

Comment:
If the checkRouting is the same as the normal routing then you could just call the `normalRoute` within here. Instead of duplicating the logic.